pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -269,6 +269,15 @@ func (mattermost *ClusterInstallation) GenerateIngress(name, ingressName string,
 			},
 		},
 	}
+	if ingressAnnotations["cert-manager.io/issuer"] != "" {
```

Comment:
thinking a bit more, I think we should not make this hardcoded. some reasons:
- if cert-manager change the annotation name (this already happened a few months ago)
- if the user has another way to generate the TLS 

so i think we need to think a bit more, and maybe have just a flag in the config file that set the whole `v1beta1.IngressTLS` and the user need to set the annotation as well otherwise will not work.

Reply:
Got it. We'll make a suggestion.

Reply:
Besides the fact that annotation names could change in the future, there is already another annotation named cert-manager.io/cluster-issuer.